x/dualstaking/keeper/keeper.go

```diff
@@ -22,6 +24,12 @@ type (
 		bankKeeper         types.BankKeeper
 		accountKeeper      types.AccountKeeper
 		epochstorageKeeper types.EpochstorageKeeper
+		specKeeper         types.SpecKeeper
+		stakingKeeper      types.StakingKeeper
+
+		delegationFS common.FixationStore
+		delegatorFS  common.FixationStore
```

Comment:
why do we need both of those?
they both show delegator -> providers
we need the opposite,
providers -> delegators like we said in the design

Reply:
`delegationFS` maps providers+chainID to delegations (key is provider_chainID_delegator; entry is delegation data).
`delegatorFS` maps delegators to their providers (key is delegator; entry is list of providers)

it's already documented in the code (in `x/dualstaking/keeper/delegate.go`), but added short comments here too (see 96a898ea)